Lesson5/Task6.cs

```diff
@@ -0,0 +1,51 @@
+namespace CSharpFundamentals.Lesson5;
+
+public class Task6
+{
+    public void StringTests()
+    {
+        try
+        {
+            Console.Write("Enter string: ");
+            string str1 = Console.ReadLine();
+
+            Console.Write("Enter string: ");
+            string str2 = Console.ReadLine();
+
+            if (String.IsNullOrEmpty(str1) || String.IsNullOrEmpty(str2))
+            {
+                throw new Exception("String is null or empty!");
+            }
+
+            Print(str1, str2);
+
+            Console.WriteLine();
+
+            bool compare = ComparingStrings(str1, str2);
+            Console.Write(compare ? "Strings are equal" : "Strings are not equal");
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e.Message);
+        }
+    }
+
+    void Print(string str1, string str2)
+    {
+        Console.Write("Entered string: ");
+        Console.Write(str1);
+        Console.WriteLine();
+        Console.Write("Entered string: ");
+        Console.Write(str2);
+    }
+
+    bool ComparingStrings(string str1, string str2)
```

Comment:
Check CompareTo method. This method must return an integer not a boolean.

Reply:
Using CompareTo Method

The CompareTo method is an instance method of a string class. It compares a value (either a string or an object) with a string instance. The return values of this method are the same as the Compare method.

The code sample in Listing 3 is an example of comparing two strings using the CompareTo method. 

// Use CompareTo method
if (author1.CompareTo(author2) == 0)
    Console.WriteLine($"Both strings have same value.");
else if (author1.CompareTo(author2) < 0)
    Console.WriteLine($"{author1} precedes {author2}.");
else if (author1.CompareTo(author2) > 0)
    Console.WriteLine($"{author1} follows {author2}.");